py-rse-index.Rmd

```diff
@@ -72,100 +125,46 @@ Sami Virtanen
 
 ### Prerequisites
 
-Learners must be able to:
+Learners must already be able to:
 
```

Comment:
I think we need to revisit the prerequisites if the Python-RSE volume is being published as a stand-alone book (i.e. we can no longer assume learners have done our novice Python course).

Reply:
Maybe something along the lines of: "Readers must already be using Python for their data analysis. They don’t need to be highly proficient, but a strong familiarity with Python syntax and basic constructs such as loops, lists conditionals (i.e. if statements) and functions is required."

Reply:
(i.e. we assume you understand the basics of python)